### PR TITLE
fix: Wisconsin cases/deaths, Fresno County cases 

### PR DESCRIPTION
**Pull request by cmcjacob**

### CODE REVIEW

**Comment by lazd** [comment]
LGTM!

**Comment by cmcjacob** [comment]
Whats the deal with the failed PR test ?

**Comment by lazd** [comment]
Hmm yeah that's odd

File: src/shared/scrapers/USA/WI/index.js [hunk 1]
```diff
@@ -176,14 +176,18 @@ const scraper = {
...
+
         regions.push({
           county: geography.addCounty(parse.string($tr.find('td:first-child').text())),
-          cases: parse.number($tr.find('td:last-child').text())
+          cases: parse.number($tr.find('td:nth-child(2)').text()),
+          deaths: parse.number($tr.find('td:nth-child(3)').text())
         });
       });
       {
+        // needs CR? not seeing this table on 3/26
```
Comment (lazd) [inline src/shared/scrapers/USA/WI/index.js]: I'm seeing 0 from this one

File: src/shared/scrapers/USA/CA/fresno-county.js [hunk 2]
```diff
@@ -11,12 +11,28 @@ const scraper = {
...
-    return {
-      cases: parse.number($('li:contains("Total cases")').text()),
-      deaths: parse.number($('li:contains("Total deaths")').text())
-    };
+  scraper: {
+    '0': async function() {
+      const $ = await fetch.page(this.url);
+      return {
+        cases: parse.number($('li:contains("Total cases")').text()),
+        deaths: parse.number($('li:contains("Total deaths")').text())
```
Comment (lazd) [inline src/shared/scrapers/USA/CA/fresno-county.js]: Hmm getting `27 cases` and `2714` cases

```
lazd-desktop:scraper lazd$ yarn timeseries -d 2020-3-23 -l "Fresno County, CA, USA"
yarn run v1.22.0
$ node src/shared/timeseries.js -d 2020-3-23 -l 'Fresno County, CA, USA'
⏳ Fetching scrapers...
✅ Fetched 128 scrapers!
✅ All scrapers are valid!
  ⚡️ Cache hit for https://www.co.fresno.ca.us/departments/public-health/covid-19 from coronadatascraper-cache/2020-3-23/40bf7ed5117f6152c41b4af585051e3e.html
⏳ Calculating ratings...
✅ Assigned ratings for 1 sources
⏳ De-duping locations...
✅ De-duped 0 locations and found 0 crosscheck failures! 
✅ Data scraped!
   - 0 cities
   - 0 states
   - 1 counties
   - 0 countries
ℹ️  Total counts (tracked cases, may contain duplicates):
   - 136 cases
   - 0 tested
   - 0 recovered
   - 0 deaths
   - 0 active
⏳ Validating and cleaning locations
✅ All locations are valid!
⏳ Fetching scrapers...
✅ Fetched 128 scrapers!
✅ All scrapers are valid!
  ⚡️ Cache hit for https://www.co.fresno.ca.us/departments/public-health/covid-19 from coronadatascraper-cache/2020-3-24/40bf7ed5117f6152c41b4af585051e3e.html
⏳ Calculating ratings...
✅ Assigned ratings for 1 sources
⏳ De-duping locations...
✅ De-duped 0 locations and found 0 crosscheck failures! 
✅ Data scraped!
   - 0 cities
   - 0 states
   - 1 counties
   - 0 countries
ℹ️  Total counts (tracked cases, may contain duplicates):
   - 186 cases
   - 0 tested
   - 0 recovered
   - 0 deaths
   - 0 active
⏳ Validating and cleaning locations
✅ All locations are valid!
⏳ Fetching scrapers...
✅ Fetched 128 scrapers!
✅ All scrapers are valid!
  ⚡️ Cache hit for https://www.co.fresno.ca.us/departments/public-health/covid-19 from coronadatascraper-cache/2020-3-25/40bf7ed5117f6152c41b4af585051e3e.html
⏳ Calculating ratings...
✅ Assigned ratings for 1 sources
⏳ De-duping locations...
✅ De-duped 0 locations and found 0 crosscheck failures! 
✅ Data scraped!
   - 0 cities
   - 0 states
   - 1 counties
   - 0 countries
ℹ️  Total counts (tracked cases, may contain duplicates):
   - 196 cases
   - 0 tested
   - 0 recovered
   - 0 deaths
   - 0 active
⏳ Validating and cleaning locations
✅ All locations are valid!
⏳ Fetching scrapers...
✅ Fetched 128 scrapers!
✅ All scrapers are valid!
  ⚡️ Cache hit for https://www.co.fresno.ca.us/departments/public-health/covid-19 from coronadatascraper-cache/2020-3-26/40bf7ed5117f6152c41b4af585051e3e.html
⏳ Calculating ratings...
✅ Assigned ratings for 1 sources
⏳ De-duping locations...
✅ De-duped 0 locations and found 0 crosscheck failures! 
✅ Data scraped!
   - 0 cities
   - 0 states
   - 1 counties
   - 0 countries
ℹ️  Total counts (tracked cases, may contain duplicates):
   - 2714 cases
   - 0 tested
   - 0 recovered
   - 0 deaths
   - 0 active
⏳ Validating and cleaning locations
✅ All locations are valid!
⏳ Fetching scrapers...
✅ Fetched 128 scrapers!
✅ All scrapers are valid!
  ⚡️ Cache hit for https://www.co.fresno.ca.us/departments/public-health/covid-19 from coronadatascraper-cache/2020-3-27/40bf7ed5117f6152c41b4af585051e3e.html
⏳ Calculating ratings...
✅ Assigned ratings for 1 sources
⏳ De-duping locations...
✅ De-duped 0 locations and found 0 crosscheck failures! 
✅ Data scraped!
   - 0 cities
   - 0 states
   - 1 counties
   - 0 countries
ℹ️  Total counts (tracked cases, may contain duplicates):
   - 27 cases
   - 0 tested
   - 0 recovered
   - 0 deaths
   - 0 active
⏳ Generating features...
✅ Found features for 1 out of 1 regions for a total of 1 features
⏳ Getting population data...
✅ Found population data for 1 out of 1 locations
⏳ Validating and cleaning locations
✅ All locations are valid!
✏️  dist/timeseries-byLocation.json written
✏️  dist/features.json written
✏️  dist/timeseries.json written
✏️  dist/locations.json written
✏️  dist/timeseries.csv written
✏️  dist/timeseries-tidy.csv written
✏️  dist/timeseries-jhu.csv written
✨  Done in 1.90s.

```

File: src/shared/scrapers/USA/WI/index.js (hunk 1 above)
Comment (lazd) [reply src/shared/scrapers/USA/WI/index.js]: `yarn timeseries 2020-3-22 -l "WI"` all zeroes for cases

Comment (cmcjacob) [reply src/shared/scrapers/USA/WI/index.js]: `yarn timeseries 2020-3-22 -l "WI, USA"` is showing only zeroes for 3/26 and 3/25 for me (probably when page changed)

**Comment by cmcjacob** [comment]
Addressed fresno county and timeseries looks correct, not getting zeroes for Wisconsin so not sure on that

**Comment by cmcjacob** [comment]
opening new PR for wisconsin only

File: src/shared/scrapers/USA/WI/index.js (hunk 1 above)
Comment (lazd) [reply src/shared/scrapers/USA/WI/index.js]: Can you fix 3/26 and 3/25 too?

**Comment by lazd** [comment]
@cmcjacob are you going to open a new PR for Fresno?

**Pull request closed without merging**